translations/en.json

```diff
@@ -807,7 +809,9 @@
     "off": "Off",
     "offline": "Offline",
     "offline-messaging": "Mailserver",
-    "offline-messaging-settings": "Mailserver settings",
+    "offline-messaging-settings": "History nodes",
```

Comment:
why do we change mailserver to history only here? we have to change "mailserver" to history node in all other places then? @errorists

Reply:
You're right @flexsurfer I'll bundle copy issues for status-react (incl 'favourites') and will propose updates to FAQ and Privacy Policy

I think this PR meets the minimal requirement to allow users to opt-out of pinging a mailserver for now